<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>outer_before</class>
 <widget class="QWidget" name="outer_before">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#outer_before
{
	background-color: rgb(255, 255, 255);
	border:1px solid black;
}</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>199</y>
     <width>1185</width>
     <height>483</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#scrollArea
{
border:1px solid black;
border-top:none;

}</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1183</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>1081</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*设置标题(包括垂直+水平的)*/
QHeaderView
{							
	
	font-size: 15px;				/*字大小*/
	font-family: '黑体';
	/*border: 1px solid #B4B4B4;*/
	border:none;
    /*border-bottom: 2px solid #999999; */		/*下边框颜色*/
    /*background: #9DC6E6;*/				/*背景浅蓝色*/ 
	min-height:30px;
	text-align: center;	       
}


/*设置标题(水平的)*/
QHeaderView::section:horizontal { 					
        border: 1px solid #AFAFAF; 			/*间隔颜色*/
	    border-bottom: 2px solid #999999; 							
        color: black;
        background: #9DC6E6;
}

/*设置鼠标停留状态*/
QHeaderView::section:horizontal:hover 
{			
        color: black;							/*字体白色*/
        background: #82AFC9;				/*背景深蓝色*/
}

/*设置鼠标按下状态*/
QHeaderView::section:horizontal:pressed 
{			
        color: black;
        background: #82AFC9;
		border-bottom: 3px solid #037BFC;				
}


QTableWidget 
{
		font-size: 14px;				/*10榜*/
		font-family: '黑体';				
		color : black;				
        border: 1px solid #000000;		
        background: white;
        gridline-color: #E0E0E0;	
        text-align: center;	
        outline:0px;			/*禁止焦点*/
}

/*设置选中的视图项*/
QTableWidget::item:selected
{				    
		border: 2px solid #0398FC;	
        color : black;				
}</string>
     </property>
     <column>
      <property name="text">
       <string>噪音源支管</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>63Hz</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>125Hz</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>250Hz</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>500Hz</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>1kHz</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2kHz</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>4kHz</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>8kHz</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>169</y>
     <width>1185</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
#widget
{
border:1px solid black;

}

QLabel
{
font-family: '黑体';
font-size: 16px;
}

QPushButton
{
font-family: '黑体';
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_info" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1185</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_info
{
	background-color: rgb(255, 255, 255);
	border:1px solid black;
}

QLabel
{
font-family: '黑体';
font-size: 16px;
}

QLineEdit
{
font-family: '黑体';
}</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>主竖区:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>甲板: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>室外编号: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>噪声限值: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>噪音源支管数量:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_zhushuqu">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_deck">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_room_number">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_noise_limit">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_duct_num">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
